code/datums/diseases/zombie.dm

```diff
@@ -99,8 +99,8 @@ var/list/zombie_cure = list()
 	else
 		if(stage >= max_stages && ishuman(affected_mob))
 			var/mob/living/carbon/human/H = affected_mob
-			H.Zombify()
 			cure()
```

Comment:
No, this is actually just a failsafe in case affected_mob disappears during Zombify() (which still shouldn't happen but w/e)